cognoml/tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+import mock
```

Comment:
Should this be

```python
from unittest import mock
```

We're using python 3.5, so `unittest.mock` will be available.